misc/dbt-materialize/CHANGELOG.md

```diff
@@ -1,7 +1,48 @@
 # dbt-materialize Changelog
+## 1.2.0 - 2022-08-31
 
-## Unreleased
+* Enable additional configuration for indexes created on view,
+  materializedview, or source materializations. Fix to use Materialize's
+  internal naming convention when creating indexes without providing
+  explicit names.
 
+  * A new optional `name` parameter:
+  ```sql
+  {{ config(materialized='materializedview',
+  indexes=[{'columns': ['col_1'], 'name':'col_1_idx'}]) }}
+  SELECT ...
+  ```
+
+  * A new `default` parameter. Default to false. If set to True, will create
+    default primary indexes.
+  ```sql
+  {{ config(materialized='materializedview',
+    indexes=[{'default': True}]) }}
+    SELECT ...
+  ```
+
+* Enable configuration of [clusters](https://materialize.com/docs/unstable/sql/create-cluster/#conceptual-framework),
+  which are a feature in a forthcoming version of Materialize Cloud, via:
+
+  * A new `cluster` connection parameter, which specifies the default cluster
+    for the connection.
+  * A new `cluster` option for materializedview and view materializations,
+    which specifies the cluster for an individual [materialized] view.
```

Comment:
I think the `[materialized]` here doesn't link anywhere!

Reply:
Also, does this option have any effect on view materializations? Is it used as the default for any indexes created on that view?

Reply:
Ah, looks like yes!

Reply:
My bad, I meant the optional [materialized] instead of a link. Updated!